plugins.cpp

```
@@ -1283,83 +1338,152 @@ void CMUSHclientDoc::OnFilePluginwizard()
+          m_AliasArray.SetAt (i, newAliasArray [i]);
+        m_AliasRevMap.swap (newAliasRevMap);
+        }
+      if (!timersToDelete.empty ())
+        m_TimerRevMap.swap (newTimerRevMap);
+
+      if (!triggersToDelete.empty ())
+        {
+        for (vector<CString>::iterator it = triggerNames.begin ();
+             it != triggerNames.end (); it++)
+          {
+          CTrigger * t;
```

Clang (14.0.6)

[warning] variable 't' is not initialized

(cppcoreguidelines-init-variables)